src/constants.h

```diff
@@ -0,0 +1,25 @@
+// sgdnet: Penalized Generalized Linear Models with Stochastic Gradient Descent
+// Copyright (C) 2018  Johan Larsson
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+#ifndef SGDNET_CONSTANTS_
+#define SGDNET_CONSTANTS_
+
+namespace sgdnet {
+  constexpr double SMALL =
+    std::pow(std::numeric_limits<double>::epsilon(), 0.75);
```

Comment:
please add a comment about what this is used for and why you need e^0.75 -- its not obvious to me